build/ninja_gen/src/python.rs

```diff
@@ -22,32 +22,32 @@ pub fn python_archive(platform: Platform) -> OnlineArchive {
     match platform {
         Platform::LinuxX64 => {
             OnlineArchive {
-                url: "https://github.com/indygreg/python-build-standalone/releases/download/20240107/cpython-3.9.18+20240107-x86_64_v2-unknown-linux-gnu-install_only.tar.gz",
-                sha256: "9426bca501ae0a257392b10719e2e20ff5fa5e22a3ce4599d6ad0b3139f86417",
+                url: "https://github.com/astral-sh/python-build-standalone/releases/download/20250517/cpython-3.13.3+20250517-x86_64_v2-unknown-linux-gnu-install_only_stripped.tar.gz",
+                sha256: "68f65b2e8098ea332ae06f2afe984f6532a42eaa9a6fc5393c4421840d8dfd70",
```

Comment:
Any reason not to use the `install_only_stripped` versions?

> An install_only_stripped archive is also available. This archive is equivalent to install_only, but without debug symbols, which results in a smaller download and on-disk footprint.

https://gregoryszorc.com/docs/python-build-standalone/main/running.html?highlight=stripped